libs/api/api_internal.h

```diff
@@ -465,6 +465,18 @@ ebpf_get_program_fd_by_id(ebpf_id_t id, _Out_ int* fd) noexcept;
 _Must_inspect_result_ ebpf_result_t
 ebpf_get_link_fd_by_id(ebpf_id_t id, _Out_ int* fd) noexcept;
 
+/**
+ * @brief Get an object's handle based on its fd.
+ *
+ * @param[in] File Descriptor of the object.
+ * @param[out] ebpf_handle_t OS based handle to the object.
```

Comment:
```suggestion
 * @param[out] handle OS based handle to the object.
```
The first token should be the argument name